bin/gke-up.sh

```diff
@@ -40,3 +41,6 @@ done
 
 # Add cert-manager
 ./deploy-cert-manager.sh
+
+# Add Prometheus
+../monitoring/bin/deploy_prometheus.sh
```

Comment:
Maybe move deploy_prometheus from the monitoring directory to forgeops/bin? I'm not sure why it would still need to be in a separate place.

Reply:
The only reason I did that was because I was a bit worried about adding another 3 scripts(monitoring/bin/...) into the /bin directory.  It's quite large as it is.  But if you don't think that's an issue then I can do.

Reply:
Personally, I don't see the point of a monitoring subdir; it would indicate that you deploy the prod sample with monitoring sometimes and without it sometimes. (There's no cert branch for example.)
To have smaller dirs, it might make sense to create some subdirs under bin and etc but I think that's a pretty good sized project.

Reply:
ok, fair point.  It could easily be removed.  I'll move the values folder under the etc folder.  I'll rename the scripts so they're explicit in their purpose